v3/util/time.go

```diff
@@ -72,6 +72,8 @@ var (
 	CABFBRs_1_8_0_Date                               = time.Date(2021, time.August, 25, 0, 0, 0, 0, time.UTC)
 	NoReservedDomainLabelsDate                       = time.Date(2021, time.October, 1, 0, 0, 0, 0, time.UTC)
 	CABFBRs_OU_Prohibited_Date                       = time.Date(2022, time.September, 1, 0, 0, 0, 0, time.UTC)
+	// Enforcement date of CRL reason codes from Ballot SC 061
+	CABFBRs_SC61_CRL_Date = time.Date(2023, time.July, 15, 0, 0, 0, 0, time.UTC)
```

Comment:
Is there a BR version number we could use for naming this date rather than the ballot name/number?  Generally zlint has used those rather than ballot codes.